### ISSUE TITLE
"New Result" score can be set to ridiculous numbers

### ISSUE THREAD

**Issue by 13XxM1CHA3lxX37** [ls1intum/Artemis#997]
### Describe the bug
just a really minor bug
The Score for applying a new result can be set to 9223372036854775295 and -9223372036854775295, while numbers larger resp. smaller return "bad request". In addition to that " Successful" field is unchecked for numbers until 99,9999999999999928945726423989989(...) and is checked for 100,000000000000007105(...)
As the score is a percentage value I think a byte or short should suffice instead of a long or whatever this datatype is or one could just restrict the input size to 3 digits. I also think the percentage value should not be able to take negative values as it could affect other scores (was not able to test the latter assumption as there is only one exercise that passed the deadline).

#### Screenshots
![Снимок экрана 2019-10-29 в 0 49 53](https://user-images.githubusercontent.com/20713070/67727233-def77900-f9e8-11e9-88c4-9195b6d4f691.png)
![Снимок экрана 2019-10-29 в 0 49 00](https://user-images.githubusercontent.com/20713070/67727239-e3239680-f9e8-11e9-8fd9-9cf47dc65a2f.png)

**Comment by thilo-behnke** [comment]
Thanks for the report.
From what I see on your screen shots, the score is very high (it should be between 0 and 100). This should not be the case because we calculate the score this way (simplified):
```
long score = maxTestScore > 0 ? successfulTestScore / maxTestScore * 100 : 0;
```

I have not seen a similar issue, so I wonder what the setup of the exercise is. Have you defined weights or set tests to only run after the due date?

**Comment by 13XxM1CHA3lxX37** [comment]
It's about the following exercises:
testcourse: https://artemis.ase.in.tum.de/#/overview/40/exercises/723 
regular course: https://artemis.ase.in.tum.de/#/overview/37/exercises/733

I don't know if you can directly access them, they are from PGdP courses.
The exercise gives 2 points
There was one test before the due date that gave one point i.e. 50% if passed and graded
After the due date a "hidden" test was executed and gave another point i.e. 100% if passed, but ungraded
After that tutors manually reviewed the submissions and could set a "new result" with a feedback that was graded

![Снимок экрана 2019-10-29 в 11 59 57](https://user-images.githubusercontent.com/20713070/67761633-f9127500-fa43-11e9-8959-5ba30e78fc0e.png)
![Снимок экрана 2019-10-29 в 12 00 53](https://user-images.githubusercontent.com/20713070/67761651-fdd72900-fa43-11e9-93dd-47b923a42a38.png)

regular course results:
![Снимок экрана 2019-10-29 в 12 07 59](https://user-images.githubusercontent.com/20713070/67762219-3e837200-fa45-11e9-8b07-adba432f4353.png)

test course results:
![Снимок экрана 2019-10-29 в 12 07 48](https://user-images.githubusercontent.com/20713070/67762196-2dd2fc00-fa45-11e9-84fa-bf3b73c2fe9d.png)

**Comment by 13XxM1CHA3lxX37** [comment]
Fixed in 3.8.7
